modules/gce_worker_group/main.tf

```diff
@@ -82,6 +81,90 @@ module "gce_workers" {
   zones                    = "${var.worker_zones}"
 }
 
+resource "google_storage_bucket" "gcloud_cleanup_archive" {
+  name    = "gcloud-cleanup-${var.env}-${var.index}"
+  project = "${var.project}"
+
+  versioning {
+    enabled = false
+  }
+
+  lifecycle_rule {
+    action {
+      type = "Delete"
+    }
+
+    condition {
+      age = "${var.gcloud_cleanup_archive_retention_days}"
+    }
+  }
+}
+
+resource "google_project_iam_custom_role" "gcloud_cleaner" {
+  role_id     = "gcloud_cleaner"
```

Comment:
Was the `cleaner` vs `cleanup` here intentional?

Reply:
Yes, with my thinking being that this is a role as opposed to an identity, but I am happy to make it `cleanup` for consistency, too :+1: